packages/solutions/app-tools/src/run/index.ts

```diff
@@ -74,7 +74,9 @@ export async function run({
   const finalConfigFile = customConfigFile || getConfigFile(configFile);
   const autoLoadPlugins = await isAutoLoadPlugins(
     appDirectory,
-    finalConfigFile,
+    finalConfigFile || 'modern.config.ts',
```

Comment:
The default value of modern.config.ts is not needed? It is already included in finalConfigFile.

Reply:
Done~